x-pack/plugin/autoscaling/src/main/java/org/elasticsearch/xpack/autoscaling/action/PolicyValidator.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.autoscaling.action;
+
+import org.elasticsearch.xpack.autoscaling.policy.AutoscalingPolicy;
+
+public interface PolicyValidator {
+    void validate(AutoscalingPolicy policy);
```

Comment:
Could you add a Javadoc for this method?

Reply:
++, added in b9d87c5676afe7cf730559423767849443df61db